packages/python/examples/metrics_django/manage.py

```diff
@@ -14,18 +12,38 @@
 def main():
     """Run administrative tasks."""
     os.environ.setdefault("DJANGO_SETTINGS_MODULE", "metrics_django.settings")
-    runserver.default_addr = "0.0.0.0"
-    runserver.default_port = os.getenv("PORT") or 8000
-    try:
+    host = "0.0.0.0"
+    port = os.getenv("PORT") or 8000
+
+    server_type = os.getenv("SERVER_TYPE", "wsgi").lower()
```

Comment:
is `SERVER_TYPE` provided by something, or are we expecting to have clients set it to a particular value?

Reply:
This variable is only used for example apps. Clients do not need to set and use it.

The purpose of this variable is just to contain 2 types of servers in one application. This way we don't need 2 identical example applications, one of which is ASGI and the other is WSGI. In general, this is used to ensure that the tests pass for both ASGI and WSGI servers.

Reply:
that makes sense, thanks